v3/src/models/codap/create-codap-document.ts

```diff
@@ -29,6 +29,7 @@ interface IOptions {
 }
 export function createCodapDocument(snapshot?: ICodapDocumentModelSnapshot, options?: IOptions): IDocumentModel {
   const { layout = "free", noGlobals = false } = options || {}
+  // Note: The version and build will not be updated after the document is first created
```

Comment:
Can/should we do something about that? Seems like the expectation would be that it would be the version that most recently saved it.

Reply:
@kswenson yeah that'd be my expectation too. 

I think we can fix it, but it seemed possibly tricky and not worth the effort right now.  If you want to prioritize it I can work on it.